managers/oclc_catalog.py

```diff
@@ -140,7 +139,7 @@ def query_bibs(self, query: str):
 
             return bibs
         except Exception as e:
-            logger.error(f'Failed to query search bibs with query {query}', e)
```

Comment:
Did you revert this because of an unexpected behavior with the exception printing? I ran into something strange but I wasn't able to isolate it, but I'm curious because I think this exception logging pattern exists elsewhere in the codebase.

Reply:
hmm lemme see

Reply:
changed it back!